AndroidStudio/assignments/advanced/3rd/DialogAssignment/app/src/main/AndroidManifest.xml

```diff
@@ -2,20 +2,16 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="jp.mixi.assignment.dialog"
     android:versionCode="1"
-    android:versionName="1.0" >
-
-    <uses-sdk
-        android:minSdkVersion="8"
-        android:targetSdkVersion="17" />
+    android:versionName="1.0">
 
     <application
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/hidey/AndroidTraining/blob/2ec9edbd5ba91b173ceff2de43f76baf77b099b0/AndroidStudio/assignments/advanced/3rd/DialogAssignment/app/src/main/AndroidManifest.xml#L7"></span>On SDK version 23 and up, your app data will be automatically backed up and restored on app install. Consider adding the attribute <code>android:fullBackupContent</code> to specify an <code>@xml</code> resource which configures which files to backup. More info: https://developer.android.com/training/backup/autosyncapi.html</td>
    </tr>
  </tbody>
</table>